src/core_plugins/kibana/public/management/sections/indices/field_filters/field_filters.html

```diff
@@ -0,0 +1,97 @@
+<h3>Source Filters</h3>
+
+<p>
+  Source filters can be used to exclude one or more fields when fetching the document source. This happens when viewing a document in the Discover app, or with a table displaying results from a saved search in the Dashboard app. Each row is built using the source of a single document, and if you have documents with large or unimportant fields you may benefit from filtering those out at this lower level.
+</p>
+
+<div ng-class="{ saving: fieldFilters.saving }" class="field-filters-container">
```

Comment:
I'd update `fieldFilters` to `sourceFilters` throughout this file as well (and in any other files where that name is used) for the sake of consistency.